reviews/models/mixins.py

```diff
@@ -182,17 +192,83 @@ def resubmission_allowed(self, ev):
         return self.reviewable.provider.reviews_workflow == workflow.Workflows.PRE_MODERATION.value
 
     def notify_submit(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        info = self.get_info()
+        context = self.get_context()
+        context['referrer'] = ev.kwargs.get('user')
+        context['template'] = 'reviews_submission_confirmation'
+        context['is_pre_moderation'] = info.get('is_pre_moderation')
+        reviews.reviews_email.send(context=context, caller='notify_submit')
 
     def notify_accept(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        self.get_accept_reject_notification()
 
     def notify_reject(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        self.get_accept_reject_notification()
 
     def notify_edit_comment(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        context = self.get_context()
+        info = self.get_info()
+        context['template'] = 'reviews_update_comment'
+        if info.get('notify_comment'):
+            reviews.reviews_email.send(context=context, caller='notify_edit_comment')
+
+    def get_info(self):
```

Comment:
I don't think this method is necessary or useful -- `notify_submit` just uses the workflow and `notify_edit_comment` just uses the `notify_comment` value (which could be moved to its own method, since it's used twice). It would be clearer to get just the needed values in the `notify_*` callbacks.